packages/traceloop-sdk/tests/test_replicate_instrumentaion.py

```diff
@@ -0,0 +1,76 @@
+import replicate
+from traceloop.sdk.decorators import workflow, task
+
+
+def test_replicate_image_generation(exporter):
+    @task(name="image_generation")
+    def generate_image():
+        image = replicate.run(
+            "stability-ai/stable-diffusion:27b93a2413e7f36cd83da926f3656280b2931564ff050bf9575f1fdf9bcd7478",
+            input={"prompt": "robots"}
+        )
+        print(image)
```

Comment:
nit: remove

Reply:
@nirga Should I remove all print statement from this test file?

Reply:
@Kartik1397 yes, these aren't printed anyway